.github/workflows/build.yml

```
@@ -0,0 +1,20 @@
+name: build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
```

Lately we switched to fetching the Go version from `go.mod`, makes it easier to update the Go version down the road :wink: Please update it in all CI actions :pray:
```suggestion
          go-version-file: 'go.mod'
```